docs/eventing/samples/sinkbinding/README.md

```diff
@@ -99,6 +99,9 @@ spec:
       apiVersion: serving.knative.dev/v1
       kind: Service
       name: event-display
+  ceOverrides:
+    extensions:
+      sink: bound
```

Comment:
does this override an exisiting extension? Or will it add (in this case) a new extension, called `sink`, with the value `bound` ?

Reply:
it will override if the event has an extension called sink, or add if it is empty.